src/views/Settings.vue

```diff
@@ -71,7 +79,8 @@ export default defineComponent({
   },
   data(){
     return {
-      baseURL: process.env.VUE_APP_BASE_URL
+      baseURL: process.env.VUE_APP_BASE_URL,
+      locales: JSON.parse(process.env.VUE_APP_LOCALES),
```

Comment:
What if we don't have an entry for locales in env or have an empty object assigned to locale in env?

Reply:
Updated the code, sir.